package.json

```diff
@@ -50,24 +50,24 @@
     "inversify": "5.0.1",
     "js-yaml": "3.13.1",
     "lodash": "4.17.15",
-    "memfs": "3.1.1",
+    "memfs": "^3.1.2",
     "moment": "2.24.0",
     "node-filter-async": "2.0.0",
-    "npm-check-updates": "4.0.1",
+    "npm-check-updates": "^4.0.4",
     "oclif": "1.15.2",
-    "qs": "6.9.1",
+    "qs": "^6.9.2",
     "reflect-metadata": "0.1.13",
     "semver": "7.1.3",
     "shelljs": "0.8.3",
-    "simple-git": "1.131.0",
+    "simple-git": "^1.132.0",
     "table": "5.4.6",
     "toposort": "2.0.2",
-    "ts-node": "8.6.2",
-    "tslib": "1.11.0",
-    "typescript": "3.8.2",
+    "ts-node": "^8.8.1",
+    "tslib": "^1.11.1",
+    "typescript": "^3.8.3",
     "update-notifier": "4.1.0",
     "xml2js": "0.4.23",
-    "yaml": "1.7.2",
+    "yaml": "^1.8.3",
```

Comment:
Remove `^` specifications. We can not allow installing relative version for the end-user.

Reply:
btw you can use `npm-check-updates` with command `ncu` to automatically update all versions

Reply:
I know, but it updates also major versions, doesn't it?

Reply:
It has an option to update just minor versions.